crates/matrix-sdk/src/notification_settings/rules.rs

```diff
@@ -276,6 +276,15 @@ pub(crate) fn get_predefined_underride_room_rule_id(
     }
 }
 
+pub(crate) fn get_predefined_underride_polls_rule_id(
```

Comment:
Would it make more sense to move this method to be under the `IsOneToOne` enum? Additionally this could also be a `From<IsOneToOne> for `PredefinedUnderrideRuleId` trait implementation, of course if it makes sense.

Reply:
I thought it was easier to read since we already have `get_predefined_underride_room_rule_id`, but I'm happy to make this change if you think it's better.

Reply:
I think it makes it easier to discover the method and the relationship between the data types, so yes please.

Reply:
Applied -> https://github.com/matrix-org/matrix-rust-sdk/pull/2644/commits/918020e6877e65eb4fc0c932f45c44a30be5e596